Source/SourceKittenFramework/library_wrapper_sourcekitd.swift

```diff
@@ -9,7 +9,7 @@ private let path = "sourcekitd.framework/Versions/A/sourcekitd"
 private let library = toolchainLoader.load(path: path)
 internal let sourcekitd_initialize: @convention(c) () -> () = library.load(symbol: "sourcekitd_initialize")
 internal let sourcekitd_shutdown: @convention(c) () -> () = library.load(symbol: "sourcekitd_shutdown")
-internal let sourcekitd_set_interrupted_connection_handler: @convention(c) (sourcekitd_interrupted_connection_handler_t) -> () = library.load(symbol: "sourcekitd_set_interrupted_connection_handler")
+internal let sourcekitd_set_interrupted_connection_handler: @convention(c) (@escaping sourcekitd_interrupted_connection_handler_t) -> () = library.load(symbol: "sourcekitd_set_interrupted_connection_handler")
```

Comment:
we shouldn't update these fixtures for Xcode 8.1 until we can run that version on Travis. (and Swift 3.0.1 on Linux).